openfisca_uk/parameters/benefits/carers_allowance/min_hours.yaml

```
@@ -0,0 +1,7 @@
+description: Minimum number of hours spent providing care
```

```suggestion
description: Minimum number of hours spent providing care to be eligible for the Carer's Allowance.
```

Or not sure if you want to mention the program, but something more explicit here may be helpful.